react/react-training/package.json

```diff
@@ -11,6 +11,7 @@
     "redux-form": "^7.4.2",
     "react-router-dom": "^4.3.1",
     "redux-thunk": "^2.3.0",
+    "riek": "^1.1.0",
```

Comment:
It isn't a good library. Let's keep it now, let's not make any changes, but let's keep it in mind for future projects